pkg/runtime/interfaces/application_configuration.go

```diff
@@ -45,6 +45,8 @@ type ApplicationConfig struct {
 	// This defines the nested path on the configured external storage provider where workflow closures are remotely
 	// offloaded.
 	MetadataStoragePrefix []string `json:"metadataStoragePrefix"`
+	// Event version to be used for Flyte workflows
+	EventVersion int `json:"eventVersion"`
```

Comment:
why is this set in flyteadmin config and not from the events source (the propeller deployment)?

Reply:
@katrogan The propeller change is already done. The eventVersion is determined per workflow. This change is essentially for all the new workflows created. All the old workflows do not have events anymore, and are already stored in Admin.

Reply:
> All the old workflows do not have events anymore.

What? What does this mean?

Reply:
Events are only sent when the workflows are not finished. So old workflows will not have events anymore sent from Propeller. This will not affect them.

Reply:
but we send events on workflow completion, right?

Reply:
Yup. We will default to 1, from the next release. @katrogan The version is pinned to the execution. So running/completing executions does not matter. Only newly created executions will be affected. Also if you revert, you are still fine